### ISSUE TITLE
[Bug?] Unable to run benchmark with Runtime Moniker like your "LinqBasic" exemple

### ISSUE THREAD

**Issue by doodz** [dotnet/BenchmarkDotNet#2470]
Hi, I'm unable to run benchmark in Net 7 or Net8 :

```ps
C:\Users\doods> dotnet --list-sdks
3.1.100 [C:\Program Files\dotnet\sdk]
5.0.408 [C:\Program Files\dotnet\sdk]
6.0.203 [C:\Program Files\dotnet\sdk]
6.0.320 [C:\Program Files\dotnet\sdk]
6.0.417 [C:\Program Files\dotnet\sdk]
7.0.114 [C:\Program Files\dotnet\sdk]
7.0.311 [C:\Program Files\dotnet\sdk]
7.0.404 [C:\Program Files\dotnet\sdk]
8.0.100 [C:\Program Files\dotnet\sdk]
```
error MSB3971
```
Enter the asterisk `*` to select all.
81

// Validating benchmarks:
// ***** BenchmarkRunner: Start   *****
// ***** Found 8 benchmark(s) in total *****
// ***** Building 2 exe(s) in Parallel: Start   *****
// ***** Done, took 00:00:08 (8.12 sec)   *****
// ***** Failed to build in Parallel, switching to sequential build   *****
// ***** Done, took 00:00:09 (9.13 sec)   *****
// Found 8 benchmarks:
//   LinqBasic.ForeachLoop: .NET 7.0(Runtime=.NET 7.0) [Count=100]
//   LinqBasic.Linq: .NET 7.0(Runtime=.NET 7.0) [Count=100]
//   LinqBasic.LinqStatic: .NET 7.0(Runtime=.NET 7.0) [Count=100]
//   LinqBasic.LinqFunk: .NET 7.0(Runtime=.NET 7.0) [Count=100]
//   LinqBasic.ForeachLoop: .NET 8.0(Runtime=.NET 8.0) [Count=100]
//   LinqBasic.Linq: .NET 8.0(Runtime=.NET 8.0) [Count=100]
//   LinqBasic.LinqStatic: .NET 8.0(Runtime=.NET 8.0) [Count=100]
//   LinqBasic.LinqFunk: .NET 8.0(Runtime=.NET 8.0) [Count=100]

Setup power plan (GUID: 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c FriendlyName: Performances élevées)
// Build Error: Standard output:

 Standard error:
 Version MSBuild 17.8.3+195e7f5a3 pour .NET
C:\Program Files\dotnet\sdk\8.0.100\Microsoft.Common.CurrentVersion.targets(1241,5): error MSB3971: les assemblys de référence de ".NETFramework,Version=v7.0" sont introuvables. Vous uti
lisez peut-être une ancienne version du kit SDK .NET pour cibler .NET 5.0 ou une version ultérieure. Mettez à jour Visual Studio et/ou votre kit SDK .NET. [C:\Users\ThibaultHERVIOU\Downloads\BenchmarkDotNet-master (1)\BenchmarkDotNet-master\samples\BenchmarkDotNet.Samples\bin\Release\net462\7020578c-1572-4ade-8483-5bc6a0be0e6c\BenchmarkDotNet.Autogenerated.csproj]      
ÉCHEC de la build.
C:\Program Files\dotnet\sdk\8.0.100\Microsoft.Common.CurrentVersion.targets(1241,5): error MSB3971: les assemblys de référence de ".NETFramework,Version=v7.0" sont introuvables. Vous uti
lisez peut-être une ancienne version du kit SDK .NET pour cibler .NET 5.0 ou une version ultérieure. Mettez à jour Visual Studio et/ou votre kit SDK .NET. [C:\Users\ThibaultHERVIOU\Downloads\BenchmarkDotNet-master (1)\BenchmarkDotNet-master\samples\BenchmarkDotNet.Samples\bin\Release\net462\7020578c-1572-4ade-8483-5bc6a0be0e6c\BenchmarkDotNet.Autogenerated.csproj]      
    0 Avertissement(s)
    1 Erreur(s)
Temps écoulé 00:00:00.60

// BenchmarkDotNet has failed to build the auto-generated boilerplate code.
// It can be found in C:\Users\ThibaultHERVIOU\Downloads\BenchmarkDotNet-master (1)\BenchmarkDotNet-master\samples\BenchmarkDotNet.Samples\bin\Release\net462\7020578c-1572-4ade-8483-5bc6a0be0e6c
// Please follow the troubleshooting guide: https://benchmarkdotnet.org/articles/guides/troubleshooting.html


```

Any idea?
 Thank doods

**Comment by timcassell** [comment]
Builds have been improved substantially since this issue was opened. Please open a new issue if you continue to run into issues.
